### PR TITLE
Buttons leave a message in chat

### PR DESCRIPTION
**Pull request by VTCobaltblood**
This PR makes buttons leave a message in chat. Really. That's all. I don't know what else to say here. Putting out a PR without a description feels dumb. But I can't give a better one.

### CODE REVIEW

**Comment by VTCobaltblood** [comment]
!review

File: code/game/machinery/buttons.dm [hunk 1]
```diff
@@ -32,6 +32,7 @@
 
 /obj/machinery/button/attack_hand(mob/living/user)
 	if(..()) return 1
+	user.visible_message("<span class='notice'>[user] hits the [src.name] button.</span>")
```
Comment (skull132) [inline code/game/machinery/buttons.dm]:
```suggestion
	user.visible_message("<span class='notice'>[user] hits [src] button.</span>")
```

Comment (VTCobaltblood) [reply code/game/machinery/buttons.dm]: Then it doesn't add thes.
image
Looks pretty ugly.

Comment (Karolis2011) [reply code/game/machinery/buttons.dm]:
```suggestion
	user.visible_message("<span class='notice'>[user] hits \the [src] button.</span>")
```

**Pull request merged**